common/views/components/SearchBar/SearchBar.tsx

```diff
@@ -62,7 +62,7 @@ const SearchBar: FunctionComponent<Props> = ({
   const defaultInputRef = useRef<HTMLInputElement>(null);
 
   return (
-    <Container>
+    <Container className="is-hidden-print">
```

Comment:
Which pages does this hide the search bar on?

If it's the global search – yeah, go for it.

If it's on search results – I’d be a bit more dubious.

But I don’t remember this component well enough to be sure.

Reply:
All of them and more!
1) The global searchbar is hidden from within the hidden Header already, but this does double hide it
2) Search result; indeed hidden - could add it as an exception if it's what we'd like!
3) Work pages, which to me and the second page type most likely to get printed and why I went for it to start.

Reply:
That's true, I think (1) and (3) are fine, especially as we don't include your search term in the search box. (Maybe I’m misremembering, but I'm sure that if you searched `X` and then clicked on a result, we'd leave `X` in the search box on the work page.)

It feels like pretty important context for search results, but I think those might need more work to be print-friendly, e.g. putting the URLs inline with the results.

Reply:
That's true; I guess the question at this stage is; are search results likely to be printed and therefore does this warrant further efforts? Just weary of how much we could do on this (and it feels like doing half a job if we don't do it 😄 ), but how much of it will be used?

(as for the search box on the works page... found a possible bug 😄 it does do that from the `/search/works` page + adds "back to search result" with prefiltered search. But if you come from `/search`, no trace of either! I'll create a backlog ticket so it gets added.)

Reply:
I would say: let's not put any work into it, but let's not make it worse than it already is – and imo removing the search box makes those pages worse.

(Good catch!)

Reply:
(https://github.com/wellcomecollection/wellcomecollection.org/issues/9756)

Reply:
My 2¢ is I feel like the search box wouldn't make a lot of sense in print under any circumstances (even if it contains the search term that you added to get you there).